concordium-node/src/p2p/peers.rs

```diff
@@ -9,7 +9,7 @@ use crate::{
     p2p::{maintenance::attempt_bootstrap, P2PNode},
     read_or_die,
 };
-
+use chrono::prelude::*;
```

Comment:
Could we be a bit more precise in importing. I think only a few definitions are needed, and this imports a lot more.